packages/react-query-observable/src/queryFn/queriesWithObservable.ts

```
@@ -1,3 +1,3 @@
-import { type Unsubscribable } from 'rxjs'
+import { type Subscription } from 'rxjs'
```

ESLint

[error] Filename is not in kebab case. Rename it to `queries-with-observable.ts`.

(unicorn/filename-case)